django_mailbox/models.py

```diff
@@ -295,6 +301,7 @@ def _get_dehydrated_message(self, msg, record):
             attachment.message = record
             for key, value in msg.items():
                 attachment[key] = value
+            attachment.cid = cid
```

Comment:
Raises exceptions of undefinied variable if ``bool(msg.get('Content-ID')) == False``.